docs/utils.md

```diff
@@ -54,3 +54,71 @@ Timeframes available for the exchange `binance`: 1m, 3m, 5m, 15m, 30m, 1h, 2h, 4
 ```
 $ for i in `freqtrade list-exchanges -1`; do freqtrade list-timeframes --exchange $i; done
 ```
+
+## List pairs/list markets
+
+The `list-pairs` and `list-markets` subcommands allow to see the pairs/markets available on exchange.
+
+Pairs are markets with the '/' character between the base currency part and the quote currency part in the market symbol.
+For example, in the 'ETH/BTC' pair 'ETH' is the base currency, while 'BTC' is the quote currency.
+For pairs traded by Freqtrade the pair quote currency is defined by the value of the `stake_currency` configuration setting,
+but using these commands you can print info about any pair/market, with any quote currency.
```

Comment:
This is odd somehow ...

maybe split this sentence at the linebreak (after configuration setting) and rephrase the 2nd one to something along the lines of

"You can print info about any pair/market with these subcommands - and you can filter by quote-currency using `--quote BTC`, or by base-currency using `--base ETH`."

Reply:
Fine, thanks 👍 

Pls review the result